examples/poker/client/web/src/context/GameContext.tsx

```diff
@@ -0,0 +1,261 @@
+        stateId,
+        ({ state }) => {
+          setPlayerState(state);
+          setConnecting(false);
+        },
+        (e) => {
+          setConnectionError(e);
+          setConnecting(false);
+        }
+      );
+      setConnection(connection);
+      return connection;
+    },
+    [token]
+  );
+
+  const disconnect = useCallback(() => {
+    if (connection !== undefined) {
+      connection.disconnect();
+      setConnection(undefined);
+      setPlayerState(undefined);
+      setEvents(undefined);
+      setConnectionError(undefined);
+    }
+  }, [connection]);
+
+  const createGame = useCallback(async () => {
+    if (token) {
+      return client.create(token, IInitializeRequest.default());
+    } else {
+      const token = (await login()) ?? "";
+
+      return client.create(token, IInitializeRequest.default());
+    }
+  }, [token]);
+
+  const joinGame = useCallback(
+    async (gameId: string) => {
+      const connection = connect(gameId);
+      await connection.joinGame({});
+    },
+    [token, connect]
+  );
+
+  const startRound = useCallback(async () => {
+    if (connection) {
+      await handleResponse(connection.startRound({}));
+    }
+  }, [connection]);
+
+  const startGame = useCallback(async () => {
+    if (connection) {
+      if (playerState?.roundStatus === RoundStatus.WAITING) {
+        await handleResponse(connection.startGame({ startingChips: 10000, startingBlind: 10 }));
+      }
+      await startRound();
+    }
+  }, [connection, playerState]);
+
+  const fold = useCallback(async () => {
+    if (connection) {
+      await handleResponse(connection.fold({}));
+    }
+  }, [token, connection]);
+
+  const raise = useCallback(
+    async (amount: number) => {
+      await handleResponse(connection?.raise({ amount }));
+    },
+    [token, connection]
+  );
+
+  const call = useCallback(async () => {
+    if (connection) {
+      await handleResponse(connection.call({}));
+    }
+  }, [token, connection]);
+
+  const endGame = () => {
+    setPlayerState(undefined);
+    connection?.disconnect();
+  };
+
+  useEffect(() => {
+    if (connectionError) {
+      toast.error(connectionError?.message);
+    }
+  }, [connectionError]);
+
+  const getUserName = useCallback(
+    (userId: string) => {
+      if (Boolean(playerNameMapping[userId])) {
+        return playerNameMapping[userId].name;
+      } else {
+        lookupUser(userId).then((response) => {
+          setPlayerNameMapping((curr) => ({ ...curr, [userId]: response }));
+        });
+        return userId;
+      }
+    },
+    [playerNameMapping]
+  );
+
+  useEffect(() => {
+    if (token) {
+      setUserInfo(HathoraClient.getUserFromToken(token));
+    }
+  }, [token]);
+
+  useEffect(() => {
+    if (playerState?.activePlayer && playerState.roundStatus === RoundStatus.ACTIVE) {
+      if (playerState?.activePlayer === user?.id) {
+        toast.success(`It's you turn`, { position: "top-center", hideProgressBar: true });
```

Comment:
```suggestion
        toast.success(`It's your turn`, { position: "top-center", hideProgressBar: true });
```

Reply:
can you make the same change to uno as well?

Reply:
should be addressed in both locations